src/style.css

```diff
@@ -1 +1,47 @@
 /* add styles here */
+body {
```

Comment:
It's not recommended to use tag names for styling (except `html` and `body`). This can lead to unexpected results when your project grows and becomes more complex. I would suggest to add a class to the `body` tag and style it instead. Refer to the checklist item: 'Don't use tag names for styling (except `html` and `body`)'